packages/starterkit/starterkit.1.0.0/opam

```diff
@@ -0,0 +1,83 @@
+# -*- conf -*-
+opam-version: "1.2"
+name: "starterkit"
+version: "1.0.0"
+maintainer: "Edgar Aroutiounian <edgar.factorial@gmail.com>"
+authors: [ "Edgar Aroutiounian <edgar.factorial@gmail.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/fxfactorial/ocaml-starterkit"
+depends: [
+  "lambda-term"
+  "ocp-indent"
+  "stringext"
+  "utop"
+  "ocp-index"
+  "merlin"
+  "ocamlfind"
+  "oasis"
+  "ocamlbuild"
+]
+
+post-messages: [
```

Comment:
I'd recommend placing the `post-messages` information in the `descr` as well, so it shows up with an `opam info`